### ISSUE TITLE
Issue with "Styling links": (broken iframe/active learning section)

### ISSUE THREAD

**Issue by ConsciousClay** [mdn/content#7312]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links

#### What information was incorrect, unhelpful, or incomplete?

![image](https://user-images.githubusercontent.com/51193876/127057710-01203e5f-4799-492b-98fe-11debfdd406b.png)

#### Specific section or headline?

Including icons on links

#### What did you expect to see?

The content disappears when clicking on the latter two links. It's not supposed to.

#### Did you test this? If so, how?

No

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/css/styling_text/styling_links`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/styling_text/styling_links/index.html
* Last commit: https://github.com/mdn/content/commit/2c4f83d81cd27fcc23d7a8e1b66521f3d9ab0bf9
* Document last modified: 2021-07-07T12:44:38.000Z

</details>

**Comment by meowmeowmeowcat** [comment]
@ConsciousClay I think this is an example for including icons on links only. The source code (HTML) of this example is
```html
<p>For more information on the weather, visit our <a href="#">weather page</a>,
look at <a href="http://#">weather on Wikipedia</a>, or check
out <a href="http://#">weather on Extreme Science</a>.</p>
```
Which all of them points to `#` or `http://#`, so they won't work when clicking on them.

**Comment by hamishwillee** [comment]
@meowmeowmeowcat The error that @ConsciousClay is raising here is not that the links don't work - they actually do trigger some navigation. The problem is that after you click those links in the box the content in the box (reproduced again) disappears - it goes blank.

![](https://user-images.githubusercontent.com/51193876/127057710-01203e5f-4799-492b-98fe-11debfdd406b.png)

**Comment by meowmeowmeowcat** [comment]
Oh, I see. Sorry for the confusion.

**Comment by meowmeowmeowcat** [comment]
Will be fixed in #7322.
